multimodal/setup.py

```diff
@@ -28,6 +28,7 @@
     "tqdm",
     "boto3",
     "requests",
+    "text-unidecode",
```

Comment:
Check the current latest version of text-unidecode in pypi and add a version cap of this package.